.flowconfig

```diff
@@ -19,6 +19,7 @@
 ; "node_modules/react-native" but in the source repo it is in the root
 .*/Libraries/react-native/React.js
 .*/Libraries/react-native/ReactNative.js
+.*/Libraries/Renderer/src/renderers/shared/utils/ReactErrorUtils.js
```

Comment:
Is this change necessary? If we don't need it with the current CI setup, why does it change when we run the tests in docker containers?